### DISCUSSION TITLE
Is the `@hyas/doks` npm package actually deprecated?

### DISCUSSION THREAD

**Discussion by salim-b** [thuliteio/doks#1060] [Q&A] [answered]
First of all: Awesome project!

Now to my question: The [Quick Start](https://getdoks.org/docs/overview/quick-start/) guide advises to use the *child theme* which depends on the [`@hyas/doks` npm package](https://www.npmjs.com/package/@hyas/doks). But on that package's site it says

> This package has been deprecated

Screenshot: 
![Screenshot 2023-05-21 at 14-36-52 @hyas_doks](https://github.com/h-enk/doks/assets/20040931/b3e1ba6d-f766-471e-963c-79397c4e4064)

Why is that? Is this project still actively developed? Or at least in maintenance mode?

**Comment by h-enk** [comment, marked as answer]
Thanks!

The package will be deprecated effectively with the release of [Doks v1.0](https://github.com/h-enk/doks/releases/tag/v1.0.0-alpha.1). I was just a bit too quick and can't seem to undeprecate it.

However, for the current stable release (Doks v0.5), the package is still valid (but thus will not be updated). The advise is then  to migrate to Doks 1.0 (when available).

**Reply by salim-b** [reply]
Thanks for the clarification!

> The advise is then to migrate to Doks 1.0 (when available).

How much effort will be required for that migration?

I just started building a new site based on the Doks child theme (stable release 0.5.0). I didn't invest much time yet adding content and customizing stuff, so I wonder: Would you recommend using the Doks v1.0.0-alpha.1 release you linked above instead? Or will I run into too much problems with that alpha and rather stick to the 0.5.0 release?

**Reply by h-enk** [reply]
> How much effort will be required for that migration?

Not sure because I haven't done it yet. But, I expect it to be pretty straightforward. A guide will be provided.

I'd recommend not using the alpha releases (v1.0.0-alpha.2 will be released tomorrow) in production, but waiting for Doks v1.0.0-beta (the only beta release coming right after v1.0.0-alpha.2).

Know that I'm pushing on releasing Doks 1.0 ASAP 💥

So, if you can't wait /just want to proceed, I'd say do that and migrate later.

**Reply by salim-b** [reply]
Thank you for the clear suggestions! And even more for all the great work you put into this project! ❤
